code/datums/mind.dm

```diff
@@ -519,7 +519,7 @@
 				if(I == src)
 					continue
 				var/mob/M = I
-				if(M.z in GLOB.z_levels_station && !M.stat)
+				if(M.z in GLOB.station_z_levels && !M.stat)

```

Comment:
Brackets around the in statement. Precedence is stupid with it

Reply:
Do you mean if((M.z in glob.station_z_levels) && !M.stat)

Reply:
yes, it is that important, since there's a very good chance byond will go "okay, that list exists, okay, M.stat is FALSE, let's continue under this if"